packages/ui-bindings/Organization/organizationInterface.tsx

```diff
@@ -16,18 +16,4 @@ fragment organizationInterface on Organization {
 ${agentInterface}
 `;
 
-export interface Organization {
-  id: number,
-  name: string,
-  type: string,
-  image: string,
-  note: string,
-  ownedEconomicResources: [number],
-  agentProcesses: [number],
-  agentPlans: [number],
-  agentEconomicEvents: [number],
-  agentCommitments: [number],
-  agentRelationships: [number],
-  agentRoles: [number],
-  agentRecipies: [number]
-}
+export interface Organization extends Agent{}
```

Comment:
This is smart! I'm gonna have to make a change to my Person interface after seeing this.

Reply:
I saw it when I was looking through the docs and I was like "sweet"